.github/scripts/pkg-test.sh

```diff
@@ -56,6 +56,18 @@ install_centos() {
   "$PKGMGR" install -y ${opts} curl nc jq || exit 1
 }
 
+install_amazon_linux() {
+  PKGMGR="$( (command -v dnf > /dev/null && echo "dnf") || echo "yum")"
+
+  pkg_version="$(echo "${VERSION}" | tr - .)"
+
+  # Install Netdata
+  "$PKGMGR" install -y /netdata/artifacts/netdata-"${pkg_version}"-*.rpm
+
+  # Install testing tools
+  "$PKGMGR" install -y ${opts} curl nc jq || exit 1
```

Comment:
```suggestion
  "$PKGMGR" install -y "${opts}" curl nc jq || exit 1
```

Reply:
And same here as in the other case.

Reply:
Please, can you add something like `# reviewdog ignore` to avoid these warnings in the future?